### PR TITLE
Mess Timings added

### LINKED ISSUE #3: display mess timings too somewhere

### PR DESCRIPTION
**Pull request by yashash1511**
image

### Also it would be nic if you could add `hacktoberfest-accepted` label to this PR or add `hactoberfest` topic to this repo.

Please do let me know if any changes are required.

Closes #3

### CODE REVIEW

**Comment by RoguedBear** [comment]
I really appreciate the contribution and the time you've taken to work on it, however i have a few change requests

- first one being, can the mess timings be displayed in the next line more subtly, something similar to like this:
  
  image
  this subtle line can also be used to display more information in the future

- can you undo the changes in `package-lock.json` and `data/db.json`?

(ps: glad to see you're in campus redeeming all those nice mess codes, but if you want to try out the main mess, feel free to use my codes for today since I am not in campus this week :upside_down_face:  )

**Comment by yashash1511** [comment]
> mess timings be displayed in the next line more subtly

will do that

> can you undo the changes in `package-lock.json` and `data/db.json`?

will do that too

> try out the main mess

already did that multiple times ;)

> I am not in campus this week

me going back home today

File: data/db.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
             {
                 "startTime": "2022-08-20 12:00:00",
                 "endTime": "2022-08-20 14:30:00",
-                "body": "{\"foodItems\":\"[{\\\"name\\\":\\\"Arbi Masala Soya Korma Rajma Sambher  Jeera Rice Roti Sirka Onion Dahi\\\",\\\"type\\\":\\\"veg\\\"}]\",\"codeUsed\":false,\"codeExpired\":true,\"multipleMealsAllowed\":false,\"mealAvailedCount\":0}",
+                "body": "{\"foodItems\":\"[{\\\"name\\\":\\\"Arbi Masala Soya Korma Rajma Sambher  Jeera Rice Roti Sirka Onion Dahi\\\",\\\"type\\\":\\\"veg\\\"}]\",\"codeUsed\":false,\"codeExpired\":true,\"multipleMealsAllowed\":false,\"mealAvailedCount\":0}"
```
Comment (yashash1511) [inline data/db.json:7]: about removing the `,` here it was causing the below error
image

**Comment by yashash1511** [comment]
Please let me know if more changes are required.

File: data/db.json (hunk 1 above)
Comment (RoguedBear) [reply data/db.json:7]: its fine, i'll let that pass

**Review by RoguedBear — APPROVED** [review]
LGTM! i'll merge it and add the hacktoberfest-accepted label

**Comment by RoguedBear** [comment]
thank you for making cool pool cool again 😎

**Pull request merged**